extensions/elasticsearch/elasticsearch-7/src/test/java/com/hazelcast/jet/elastic/ElasticSupport.java

```diff
@@ -29,8 +29,12 @@
 
 public final class ElasticSupport {
 
-    public static final DockerImageName ELASTICSEARCH_IMAGE = DockerImageName.parse("elasticsearch:7.10.1")
+    public static final String TEST_ELASTIC_VERSION = System.getProperty("test.elastic.version", "7.17.7");
```

Comment:
Nice, we can check elastic support for different versions (if needed) as we do with Kafka, FYI @olukas

Reply:
@ldziedziul we can do that. Could you please create Jira for it?

Reply:
Created https://hazelcast.atlassian.net/browse/HZ-1752